pkg/shell/machines.js

```diff
@@ -584,6 +584,17 @@ define([
         "#024848"
     ];
 
+    module.colors.parse = function parse_color(input) {
+        var div = document.createElement('div');
+        div.style.color = input;
+        var style = window.getComputedStyle(div, null);
+        for (var i = 0; i < style.length; i++) {
+            if (style[i] == "color")
+                return style.getPropertyValue(style[i]);
+        }
+        return div.style.color;
```

Comment:
Why do you go through the properties of `style`? If I understand correctly, `style.color` should always exist in the object returned from `getComputedStyle`.